lib/plugins/filter/meta_generator.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const rMetaGenerator = /<meta\s+name=['|"]?generator['|"]?/i;
```

Comment:
Current code will recognize incorrect expressions as html.

``` html
<meta name="generator'>
```

Also, "|" is described in the character set, this expression will detect `<meta name=|generato |>`. Is this just as intended?

Reply:
I prepared regExp which seems to be able to express the intention of writing regExp.
`/meta\s+name(["']?)generator\1/i`

Reply:
@segayuu Oops, didn't notice that, I took that regExp from [hugo](https://github.com/gohugoio/hugo/blob/master/transform/hugogeneratorinject.go#L24). I'll make a tweak. I'm considering to use cheerio to detect this meta tag.